imports/plugins/core/revisions/server/startup.js

```diff
@@ -0,0 +1,175 @@
+import { Products, Revisions } from "/lib/collections";
+import { Logger } from "/server/api";
+import { diff } from "deep-diff";
+
+
+Products.before.insert((userId, product) => {
```

Comment:
I think hooks should move to `server/hooks/hooks.js` for more consistent implementation of hooks.  Is there a specific reason it's in startup.js?